.github/workflows/Build.yml

```diff
@@ -32,6 +32,11 @@ jobs:
       with:
         dotnet-version: '3.1.x'
 
+    - name: Install .NET Core 5.0
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.101'
+
```

Comment:
Can we stop installing .Net Core 3.1? And what about using a float number for the patch version?

Reply:
Absolutely. Done